Assets/VRTK/Scripts/Utilities/ObjectFollow/VRTK_TransformFollow.cs

```diff
@@ -93,6 +93,7 @@ protected virtual void OnDisable()
         {
             transformToFollow = null;
             transformToChange = null;
+            positionToFollow = Vector3.zero;
```

Comment:
This is not needed and actually changes the setting a user set. The cleanup in this method is only supposed to stop listening to events and drop any references that are set up again in `OnEnable`.

Reply:
Right.